src/benchmark.cc

```diff
@@ -781,7 +798,14 @@ void RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
         report.cpu_accumulated_time = cpu_accumulated_time;
         report.bytes_per_second = bytes_per_second;
         report.items_per_second = items_per_second;
+        report.arg1 = b.arg1;
```

Comment:
Yes, on the function ComputeBigO() on the file reporter.cc to populate the vector "n". I need arg1 and arg2 to know the length of the test. currently I am only using arg1, because I only use one variable, but in the near future I will use also arg2. This will let me calculate, for example, the time of string concatenation as a function of the length of both strings (N+M or in this case arg1 and arg2), but currently the bigO can only be calculated as a function of arg1.